backend-lambda/services/productService.js

```diff
@@ -68,4 +68,21 @@ const createProduct = async (pool, product) => {
   return res?.rowCount;
 };
 
-export default { getAllProducts, updateProduct, createProduct };
+const deleteProduct = async (pool, id) => {
+  // Check pool for object type
+  poolCheck(pool);
+
+  // Format query
+  const query = {
+    name: "deleteProduct",
+    text: `DELETE FROM product WHERE product_id = $1`,
+    values: [id],
```

Comment:
As it stands, id is a string and not a number. Does this cause any issues with the sql?

Reply:
And if it doesn't, you should update your tests so that `id` is a string there as well.

Reply:
oh interesting, I thought ID was a serialized integer?

![image](https://github.com/user-attachments/assets/141148fb-8efb-4c85-9287-90f26bdc1e30)

Reply:
Oh is it that ID is being passed as a string in the event?

Reply:
Correct